src/ott/initializers/linear/initializers.py

```diff
@@ -96,13 +103,17 @@ class DefaultInitializer(SinkhornInitializer):
   """Default initialization of Sinkhorn dual potentials/primal scalings."""
 
   def init_dual_a(
-      self, ot_prob: 'linear_problem.LinearProblem', lse_mode: bool
+      self,
+      ot_prob: 'linear_problem.LinearProblem',
```

Comment:
Since we no longer need the `TYPE_CHECKING`, you you remove the `'` around `linear_problem.LinearProblem`?